wc-admin.php

```diff
@@ -92,6 +92,16 @@ function deactivate_wc_admin_plugin() {
 }
 register_deactivation_hook( WC_ADMIN_PLUGIN_FILE, 'deactivate_wc_admin_plugin' );
 
+/**
+ * Update the database tables if needed. This hooked function does NOT need to
+ * be ported to WooCommerce's code base - WC_Install will do this on plugin
+ * update automatically.
+ */
+function wc_admin_init() {
+	WC_Admin_Api_Init::create_db_tables();
```

Comment:
Is there any issue with running this on every `init`? Or is this done in woo core too?

Reply:
It's not a long term solution. We won't want to merge this bit to core (hence the comment). I can open an issue to specifically remove this then.

FWIW, core has the same problem - dbdelta is only run on version updates.

I just had another idea - what if we test for some define (e.g. WC_ADMIN_DEV) and run it on init only when that is present - i.e. just in case this accidentally makes it into the core merge.

Reply:
Yeah I like that idea of adding a sanity check - maybe just do WP_DEBUG check. I figure most beta testers will never hit this issue - just those of us that are swapping out git branches often.

Reply:
Fixed in https://github.com/woocommerce/wc-admin/pull/747/commits/83f9a1859cdc2721f063a0c58c216780525d7848